### PR TITLE
Use CFPB brand colors in cf-icons component

### PR DESCRIPTION
**Pull request by cfarm**
Adds CFPB brand colors as the default variable values for colors in `cf-icons`. 

## Testing

In this repo:
- check out this branch
- run `npm run cf-link`

In a second local clone of this repo in a different folder:

- pull down and check out the `gh-pages-rm-overrides` branch: `git fetch && git checkout gh-pages-rm-overrides`
- if you haven't already in this second clone, [run all the install steps for the project](https://github.com/cfpb/capital-framework/tree/gh-pages#installation)
- run `npm run cf-link`
- run `npm run build`
- run `npm start`
- Compare colors on local site: 
   - http://localhost:3000/components/cf-icons/ 
   - to the live cf documentation to make sure they match: https://cfpb.github.io/capital-framework/components/cf-icons/

## Notes

-

## Todos

-

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: src/cf-icons/usage.md [hunk 1]
```diff
@@ -37,11 +37,11 @@ Overwrite them in your own project by duplicating the variable `@key: value`.
 
 ### Color variables
 
-`$color-` variables are from 18F's
-[U.S. Web Design Standards](https://github.com/18F/web-design-standards/blob/staging/src/stylesheets/core/_variables.scss)
+Color variables referenced in comments are from [cf-core cf-brand-colors.less](https://github.com/cfpb/capital-framework/blob/master/src/cf-core/src/cf-brand-colors.less).
```
Comment (anselmbradford) [inline src/cf-icons/usage.md]: Is this a URL that _will_ be present? It's 404 for me currently.

**Comment by anselmbradford** [comment]
The color gray is slightly different (from #5b616b to #5a5d61), but this is expected, right?

File: src/cf-icons/usage.md (hunk 1 above)
Comment (cfarm) [reply src/cf-icons/usage.md]: yup, once the cf-core component is released, that will be the link to the cf-core vars on the master branch.

**Comment by cfarm** [comment]
I guess the gray coming through before was the 18F color (it should have been our gray, from the cf-theme-overrides.less file). But 5a5d61 is correct.

**Pull request merged**